azkaban-web-server/src/main/java/cloudflow/services/ExecutionServiceImpl.java

```diff
@@ -192,10 +198,74 @@ public JobExecution getJobExecution(String executionId, String jobDefinitionId,
       attempts.add(new JobExecutionAttempt(id, startTime, endTime, status));
     }
 
-    JobExecutionAttempt lastAttempt =
-        new JobExecutionAttempt(attempts.size(), jobNode.getStartTime(), jobNode.getEndTime(),
-            jobNode.getStatus());
+    JobExecutionAttempt lastAttempt = new JobExecutionAttempt(attempts.size(),
+        jobNode.getStartTime(), jobNode.getEndTime(), jobNode.getStatus());
     attempts.add(lastAttempt);
     return attempts;
   }
+
+  @Override
+  public String createExecution(ExecutionParameters executionParameters) throws CloudFlowException {
+    requireNonNull(executionParameters);
+    // TODO ypadron: check user permissions
+
+    // TODO ypadron: obtain flow and project dynamically
+    Project project = this.projectManager.getProject("yb-test");
```

Comment:
Now that project related code has been checked in, can you check if project can be fetched dynamically?

Reply:
It's possible to retrieve projects now but for creating executions what we have is a flow definition id not a project id, we'll be fetching projects by one of their flows.